sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/sweep.py

```diff
@@ -57,53 +57,49 @@
 
 
 class Sweep(ParameterizedSweep, BaseNode):
-    """Base class for sweep node, used for command component version consumption.
+    """Base class for sweep node, used for command component version
+    consumption.
```

Comment:
The docstring's short description should be on one line: https://peps.python.org/pep-0257/#multi-line-docstrings
```suggestion
    """Base class for sweep node, used for command component version consumption.
```